charts/kube-prometheus-stack/templates/prometheus/servicemonitors.yaml

```
@@ -21,6 +21,15 @@ items:
     {{- if .jobLabel }}
       jobLabel: {{ .jobLabel }}
     {{- end }}
+    {{- if .metricRelabelings }}
+          metricRelabelings:
+{{ toYaml .metricRelabelings | indent 12 }}
```

Could you switch to the following syntax? It may make the template more readable

```suggestion
            {{- toYaml .metricRelabelings | nindent 12 }}
```

and do it for the other way well. Thanks!